packages/kbn-esql-editor/src/history_local_storage.ts

```diff
@@ -77,15 +80,7 @@ export const addQueriesToCache = (
     });
   }
 
-  const queriesToStore = getCachedQueries();
-  const localStorageQueries = getHistoryItems('desc');
-  // if the user is working on multiple tabs
-  // the cachedQueries Map might not contain all
-  // the localStorage queries
-  const newQueries = localStorageQueries.filter(
```

Comment:
ℹ️ this is an unnecessary code. I should have cleaned it up when I removed the last duration column but I missed that.